src/github/client/app-token-holder.ts

```diff
@@ -37,17 +47,21 @@ export default class AppTokenHolder {
 	/**
 	 * Gets the current app token or creates a new one if the old is about to expire.
 	 */
-	public getAppToken(): AuthToken {
-		if (!this.currentToken || this.currentToken.isAboutToExpire()) {
-			this.currentToken = AppTokenHolder.createAppJwt(this.key, this.appId);
+	public getAppToken(appId: InstallationId): AuthToken {
```

Comment:
This is the main change in `AppTokenHolder`: previously it held a single app token, because we had a single app. With each GHE instance having its own app, it now needs to hold a token for each of those apps. So we now have to pass in an installation ID to get the right app token.